_data/authors.yml

```diff
@@ -84,13 +100,41 @@ linda_xiang:
   avatar: linda_xiang.svg
   github: lindaxiang
 
+mitchell_shiell:
+  name: Mitchell Shiell
+  position: Outreach Lead
+  info: Scientist, educator, and digital creative using his powers to increase public support and awareness of incredible and innovative technologies. Also an avid hockey player and passionate photographer.
+  avatar: mitchell_shiell.svg
+  github: mitchellshiell
+
```

Comment:
Azher Ali_Mohammed:
  name: Azher Ali Mohammed
  position: Software Developer
  info: An engineer who loves to solve complex coding and work on logic building. Hold background in Computer Science and Master of Science in Project Management and Analysis.  Also seeks interest in natural and wild life photography.
  avatar: Azher.svg
  github: Azher2Ali